tests/E2E Tests/WebAppUiTests/TestingWebAppLocally.cs

```diff
@@ -103,5 +103,86 @@ public async Task ChallengeUser_MicrosoftIdFlow_LocalApp_ValidEmailPassword()
             playwright.Dispose();
         }
     }
+
+    [Fact]
+    [SupportedOSPlatform("windows")]
+    public async Task ChallengeUser_MicrosoftIdFlow_LocalApp_ValidEmailWithCiamPassword()
+    {
+        // Arrange
+        const string _clientSecret = "...";
+        Process? process = null;
+        const string TraceFileName = TraceFileClassName + "_ValidEmailPassword";
+        using IPlaywright playwright = await Playwright.CreateAsync();
+        IBrowser browser = await playwright.Chromium.LaunchAsync(new() { Headless = false });
```

Comment:
Probably best to default headless to true so tests run faster and there aren't browsers popping up when people run Id Web's tests.

Reply:
For sure, this probably just a forgotten setting.

Reply:
Yes, this is for testing. will be changed back